net-utils/src/ip_echo_server.rs

```diff
@@ -173,7 +173,11 @@ pub fn ip_echo_server(
 ) -> IpEchoServer {
     tcp_listener.set_nonblocking(true).unwrap();
 
-    let runtime = Runtime::new().expect("Failed to create Runtime");
+    let runtime = tokio::runtime::Builder::new_multi_thread()
+        .thread_name("solIpEchoSrvrRt")
+        .enable_all()
+        .build()
+        .unwrap();
```

Comment:
Keep the expect instead?

Reply:
I'm not a huge fan of those expects. To me, is is comparable to how we just `.unwrap()` every time we try to acquire a `RwLock` or `Mutex`.

Even more proper would be bubbling an error up and halting the validator; this would be simple enough to do and I can tackle that. But, I would rather do that separate from this PR

Reply:
the value of `expect()` over `unwrap()` is justifying the reason that you believe it is safe to panic on failure. we should _always_ prefer `expect()` to `unwrap()`. no undefended panicking

Reply:
> the value of expect() over unwrap() is justifying the reason that you believe it is safe to panic on failure

I don't think the pre-existing message of `Failed to create Runtime` conveys your sentiment much (if any) better than `.unwrap()`. In this case, we're trying to allocate a resource and to me, the `.unwrap()` implicitly says the same thing of "bail out if we can't get this"

To be clear, I do the see the value in other cases. For example, here is a little snippet from Blockstore. The below `.expect()` message is more informative than a regular `.unwrap()`:
```rust
            .checked_add(1)
            .expect("overflow from trusted value");
```

In any case, I restored the `expect()`'s that I changed to `unwrap()`, and did two bonus `unwrap() ==> expect()` conversions for the Rayon pools in this PR. In that sense, I have not made anything "worse", and I think any additional cleanup can be considered separately.